odoo/addons/base/tests/test_image.py

```diff
@@ -11,19 +12,23 @@
 from odoo.tests.common import TransactionCase
 
 
+def img_open(data):
+    return Image.open(io.BytesIO(data))
+
+
 class TestImage(TransactionCase):
     """Tests for the different image tools helpers."""
     def setUp(self):
         super(TestImage, self).setUp()
         self.bg_color = (135, 90, 123)
         self.fill_color = (0, 160, 157)
 
-        self.base64_1x1_png = b'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAADElEQVR4nGNgYGAAAAAEAAH2FzhVAAAAAElFTkSuQmCC'
-        self.base64_svg = base64.b64encode(b'<svg></svg>')
-        self.base64_1920x1080_jpeg = tools.image_to_base64(Image.new('RGB', (1920, 1080)), 'JPEG')
+        self.img_1x1_png = base64.b64decode(b'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAIAAACQd1PeAAAADElEQVR4nGNgYGAAAAAEAAH2FzhVAAAAAElFTkSuQmCC')
+        self.img_svg = b'<svg></svg>'
+        self.img_1920x1080_jpeg = base64.b64decode(tools.image_to_base64(Image.new('RGB', (1920, 1080)), 'JPEG'))
```

Comment:
`base64.b64decode(tools.image_to_base64` looks useless, could use `image_apply_opt` instead to just transform to JPEG without base64?

Reply:
yes, good catch